test/truffleruby-tool/lib/activesupport_exclusions.yaml

```diff
@@ -1,3 +1,6 @@
 ---
 TimeExtMarshalingTest:
 - test_marshalling_preserves_fractional_seconds
+FileStoreTest:
+- test_filename_max_size
```

Comment:
What changed in 2.3.7 to necessitate this?

Reply:
I think the behaviour of `tempfile` changed, part of the security mitigations.